[2025-03-31T08:34:02.567+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T08:34:02.621+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.second_task scheduled__2025-03-27T00:00:00+00:00 [queued]>
[2025-03-31T08:34:02.642+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.second_task scheduled__2025-03-27T00:00:00+00:00 [queued]>
[2025-03-31T08:34:02.645+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-31T08:34:02.679+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-03-27 00:00:00+00:00
[2025-03-31T08:34:02.695+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2562) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-31T08:34:02.699+0000] {standard_task_runner.py:72} INFO - Started process 2574 to run task
[2025-03-31T08:34:02.695+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'second_task', 'scheduled__2025-03-27T00:00:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmprwwkd8va']
[2025-03-31T08:34:02.704+0000] {standard_task_runner.py:105} INFO - Job 84: Subtask second_task
[2025-03-31T08:34:02.817+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v4.second_task scheduled__2025-03-27T00:00:00+00:00 [running]> on host 9663692256f6
[2025-03-31T08:34:03.008+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T00:00:00+00:00'
[2025-03-31T08:34:03.009+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T08:34:03.037+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-31T08:34:03.038+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will execute after task1!']
[2025-03-31T08:34:03.049+0000] {subprocess.py:99} INFO - Output:
[2025-03-31T08:34:03.052+0000] {subprocess.py:106} INFO - hey, I am task2 and will execute after task1!
[2025-03-31T08:34:03.053+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-31T08:34:03.088+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T08:34:03.090+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=second_task, run_id=scheduled__2025-03-27T00:00:00+00:00, execution_date=20250327T000000, start_date=20250331T083402, end_date=20250331T083403
[2025-03-31T08:34:03.162+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T08:34:03.176+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
